<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zhuce</class>
 <widget class="QWidget" name="zhuce">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>191</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEditAccount">
      <property name="cursor">
       <cursorShape>SizeVerCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid gray; border-radius: 5px; padding: 2px;</string>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditPassword">
      <property name="styleSheet">
       <string notr="true">border: 1px solid gray; border-radius: 5px; padding: 2px;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditRePassword">
      <property name="styleSheet">
       <string notr="true">border: 1px solid gray; border-radius: 5px; padding: 2px;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditName">
      <property name="styleSheet">
       <string notr="true">border: 1px solid gray; border-radius: 5px; padding: 2px;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditIdCard">
      <property name="styleSheet">
       <string notr="true">border: 1px solid gray; border-radius: 5px; padding: 2px;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="styleSheet">
       <string notr="true">border: 1px solid gray; border-radius: 5px; padding: 2px;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="zhuceBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border-radius:4px;background:#409eff;color:white;font-size:14px}</string>
        </property>
        <property name="text">
         <string>注册</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border-radius:4px;background:#409eff;color:white;font-size:14px}</string>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>0</y>
     <width>381</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Image/Imageres/BackGround3.jpg);
border-image: url(:/Image/Imageres/BlueSkyAri.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注册账号</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
